### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by ZuhRa-HashiMi**
Use basic JavaScript syntax.
Use JavaScript to manipulate DOM elements.
Use JavaScript events.

For this milestone of my  portfolio website, I implemented the mobile menu using your recently acquired JavaScript knowledge.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    burger.classList.toggle('toggle');
+  });
+
+  document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+    nav.classList.remove('nav-active');
+    burger.classList.remove('toggle');
+  }));
+};
+
+navSlide();
```
Comment (thecodechaser) [inline js/script.js]: - [ ] When I click on the mobile menu options, the drop-down menu doesn't disappear. Please make sure when the user clicks any of the mobile menu options, the mobile menu disappears and the correct part of the page is displayed.

image

Comment (thecodechaser) [inline js/script.js]: - [ ] Please use the correct color `#67798E` for the close(X) button.

### Your design
image

### Figma design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ZuhRa-HashiMi, and @io-oseinana👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @ZuhRa-HashiMi        :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**